### ISSUE TITLE
ConnectAnyThing doesn't activate gpio

### ISSUE THREAD

**Issue by glennhendrick** [IntelOpenDesign/ConnectAnyThing#5]
I've been trying to use the CAT image to control a relay board.  The webpage is there but it doesn't seem to do anything on the actual boar.  I also tried manually activation the GPIO thru the sysFS but get no action.  I didn't see anything on the relay board using pins 4 - 7 so I tried the onboard GPIO13 LED and still see nothing.  Any ideas?

**Comment by lucasainsworth** [comment]
Hi Glenn,
If you get the webpage but the board pins aren't responding, I would guess that there's something wrong with the CAT Arduino sketch...  did you upload anything to the board after you installed the SD image?  CAT currently works using an Arduino sketch that communicates with the webpage, so overwriting the sketch would produce the problem you describe.  You also might see this problem if you're using a GEN1 Galileo that hasn't had a firmware update.  If you're using a GEN2, or GEN1 with updated firmware, I'd try re-installing the CAT SD image first, and going from there.  Good luck!

**Comment by glennhendrick** [comment]
I didn't see anything about a sketch in any of the docs.  The SD card image is the only software I saw that I was being told to download.  I'm using a Gen1 but the firmware is up to date.  Can you point me to the sketch?  Did I miss that in the instructions? Also,  The webpage comes up squished into my phone's screen.  All of the words and buttons are on top of each other.  Is that just me or is this an issue that's been seen before?
Thanks for your help.  I'm looking forward to working with CAT and Makenode after that.  Our Makerspace is opening for the first time in a few days and I was hoping to bring a Fwooshball table.

Date: Mon, 19 Jan 2015 10:23:50 -0800

**Comment by lucasainsworth** [comment]
Are you using Connect AnyThing v0.2.1?  GEN2 Galileos use a different pin-mapping, so we had to change the pin mappings internally in the latest versions of CAT. 
try this one: https://github.com/IntelOpenDesign/ConnectAnyThing/releases/tag/v0.2.1

Don't worry about the sketch, it's all included.  It's only a problem if you try to upload a new sketch to a board running CAT.

As for page scaling, that's usually something that changes device-to-device.  You can try pinch-zooming out on a mobile device, or control + "-" on pc.  For this version of CAT, we recommend using an Android device if you have one, and the Dolphin web browser. -https://play.google.com/store/apps/details?id=mobi.mgeek.TunnyBrowser&hl=en

**Comment by glennhendrick** [comment]
I'll try v 2.1.  The docs say 2.2 and below.  I'm using a nexus 5 with lollipop.  I'll try dolphin.  Does the sketch load with the os?

--- Original Message ---

**Comment by lucasainsworth** [comment]
Yes, once the Galileo finishes startup, it looks for sketch.elf and runs it.  The sketch is already included in the CAT image.  That's why if you upload your own sketch using the Arduino IDE, it overwrites the CAT sketch, but everything else (webpage, hotspot, etc.) still works.

**Comment by glennhendrick** [comment]
Now I've got it working on a breadboard, but not with the shield I'm using.  
Thanks.
Dolphin works better, but I still can't log in from my Chromebook.  
Any settings to look for?

**Comment by glennhendrick** [comment]
Great.  Thanks for all of the help.  Login is still glitchy.  It works best with Dolphin.  I can't get on with Chrome at all.  On my Android I have to go to "Sign in to Network" to use the buttons.  Chrome looks for a connection and never makes it.
